tests/v3/index.test.ts

```diff
@@ -52,6 +52,8 @@ describe("types", () => {
     };
     expect(swaggerToTS(schema)).toBe(
       format(`
+      export interface operations {}
```

Comment:
🤔 at first I was going to ask to not omit this if empty, but after thinking about it, I think I like always having it there, rather than being conditionally there or not. Seems safer to always omit